prime-router/metadata/schemas/covid-19.schema

```diff
@@ -767,9 +788,12 @@ elements:
     natFlatFileField:
     hl7Field: MSH-3
 
-  - name: specimen_collection_site # collection location
-    type: TEXT # CWE without a defined table
-    hl7Field: SPM-10
+  - name: sending_facility
+    type: HD
+    # Default is the official name of PRIME from CDC PHIN. Matches the OID in sending_application_id, below.
```

Comment:
Yes, this seems redundant with sending_application.   Should we just use a `mapper` to generate this?

Reply:
I wasn't sure if this was ever going to change. For example, if we batch up results from a single facility we might want to fill in this field.

Reply:
@mauricereeves, check with @RickHawesUSDS .